element_calcium_imaging/imaging_no_curation.py

```diff
@@ -1532,7 +1532,17 @@ def make(self, key):
         params, method = (ProcessingParamSet * ProcessingTask & key).fetch1(
             "params", "processing_method"
         )
-        caiman_params = (ProcessingTask * ProcessingParamSet & key).fetch1("params")
+        if "indicies" in params:
+            caiman_params = {
+                "motion": {
+                    "indicies": (
+                        slice(*params["indicies"][0]),
+                        slice(*params["indicies"][1]),
+                    )
+                }
+            }
```

Comment:
I don't think you need to nest this inside `motion`

Reply:
I did it this way based on my understanding of the documentation [here](https://caiman.readthedocs.io/en/master/Getting_Started.html) under the `Parameters` subheading. Let me know if you think it is still not necessary to nest the parameters like that

Reply:
and this implementation would discard anything in `params`, replacing it with just the `motion` part